src/control/cmd/dmg/system.go

```diff
@@ -178,6 +178,7 @@ func (cmd *systemEraseCmd) Execute(_ []string) error {
 type systemStopCmd struct {
 	baseRankListCmd
 	Force bool `long:"force" description:"Force stop DAOS system members"`
+	Full  bool `long:"full" description:"Attempt a full shutdown of DAOS system"`
```

Comment:
doc only: can we add some warning here like do not use in production or something along that line?

Reply:
Sure, what about "Experimental, not for use in production"?